monero-outreach-docs/translations/es/quickfacts/QuickFacts_es.md

```diff
@@ -121,8 +122,9 @@ A pesar de que Monero ya está disponible y está siendo usado a nivel global, l
 + **_[Análisis SWOT de Monero]()_**
 + **_[Conectarse con la Comunidad de Monero](https://getmonero.org/community/hangouts/)_**
```

Comment:
This link is missing to be changed:

https://www.getmonero.org/es/community/hangouts/